piglow/piglow.go

```diff
@@ -34,29 +30,24 @@ func (p *PiGlow) Enable() error {
 	return p.conn.Write([]byte{0x00, 0x01})
 }
 
-// Setup enables normal operations, resets the internal registers, and enables
+// setup enables normal operations, resets the internal registers, and enables
 // all LED control registers
-func (p *PiGlow) Setup() error {
```

Comment:
How would you feel about removing this part from the PR while we discuss the proper way it should be called/named.